atomic_reactor/utils/koji.py

```diff
@@ -296,22 +297,28 @@ def get_koji_module_build(session, module_spec):
     return build, rpm_list
 
 
-def get_buildroot(build_id, tasker, osbs, rpms):
+def get_buildroot(build_id, osbs, rpms):
     """
     Build the buildroot entry of the metadata.
     :param build_id: str, ocp build_id
-    :param tasker: ContainerTasker instance
     :param osbs: OSBS instance
     :param rpms: bool, get rpms for components metadata
     :return: dict, partial metadata
     """
-    docker_info = tasker.get_info()
-    host_arch, docker_version = get_docker_architecture(tasker)
+    # OSBS2 TBD
+    # docker_info = tasker.get_info()
+    podman_info = None  # podman info
+    # OSBS2 TBD
+    host_arch = platform.processor()
```

Comment:
I realize that this is TBD, but is there any specific reason to use [platform.processor](https://docs.python.org/3/library/platform.html#platform.processor) rather than [platform.machine](https://docs.python.org/3/library/platform.html#platform.machine)?

platform.machine is used here btw https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/plugins/post_export_operator_manifests.py#L23

Reply:
post_tag_and_push.py and build_orchestrate_build.py are using platform.processor

Reply:
IC, well it doesn't matter much anyway